### ISSUE TITLE
REST API: Add option to unpack uploaded zipfile on resource creation

### ISSUE THREAD

**Issue by selimnairb** [hydroshare/hydroshare#485]
Calls to POST /hsapi/resource/ should be able to specify that the uploaded resource file is a zipfile archive that should be unpacked as part of the resource creation process

**Comment by selimnairb** [comment]
Once #924 is merged, we will need to add support to the REST API for: (1) specifying unzipping on resource creation; and (2) checking the status of unzipping.

**Comment by dtarb** [comment]
@alvacouch please consider the best solution to this functionality in API work

**Comment by dtarb** [comment]
@sblack close if done.  Act on if not.

**Comment by sblack-usu** [comment]
Workflows with hsclient and czhub create a resource without uploading files.  Files are uploaded and unzipped after creation.  This functionality is no longer needed.
